processor/src/main/java/org/mapstruct/ap/internal/util/FormattingMessager.java

```diff
@@ -67,4 +67,10 @@ void printMessage(Element e,
                       AnnotationValue v,
                       Message msg,
                       Object... args);
+
+    /**
+     * Just log as plain note
+     * @param log
+     */
+    void log(int level, String log);
```

Comment:
Perhaps name it `verbose`?

Reply:
> Some other thing, all other places where we do logging with use our [`Message`](https://github.com/mapstruct/mapstruct/blob/master/processor/src/main/java/org/mapstruct/ap/internal/util/Message.java). Why not do it the same? We can even keep everything as is and only output `NOTE` when the flag is enabled, otherwise it is a noop. Maybe even outputting the annotation value and element could have more output.

I can try to fit everything in a standard terminology (Message will enforce that).. I tried to use terms as create, select etc..

Reply:
The terms were really good. Forgot to mention that 😀

Reply:
> The terms were really good. Forgot to mention that 😀

Thanks...